### ISSUE TITLE
log and pth file for training

### ISSUE THREAD

**Issue by zhaiyutong** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#103]
Hi, thank you so much for your paper and your code!

I am doing a comparison between your method and mine, but I don't have enough space and time for retraining your model. 

Could you share your log file for training? And with .pth file for each 5000 iteration would be better!

Thanks a lot!

**Comment by Daniil-Osokin** [comment]
Hi! For a fair comparison I suggest to use numbers from the paper. We have used Caffe to train the network and obtained the best results with it. This repository is reimplementation in PyTorch and accuracy differs due to different frameworks. For the Caffe version there are no artifacts available.

**Comment by Daniil-Osokin** [comment]
Hope, it is clear now.
